services-geojson/build.gradle

```
@@ -1,12 +1,89 @@
-apply plugin: 'java-library'
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFrameworkConfig
 
-dependencies {
-  // Gson
-  api libs.gson
+plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.kotlinx.serialization)
+    alias(libs.plugins.kotlin.cocoapods)
+    alias(libs.plugins.kotlin.dokka)
+}
+
+kotlin {
+    jvmToolchain(17)
+
+    jvm()
+
+    def xcf = new XCFrameworkConfig(project)
+    [iosX64(), iosArm64(), iosSimulatorArm64()].forEach { target ->
+        target.binaries.framework { framework ->
+            baseName = "maplibre-geojson"
+            xcf.add(framework)
+        }
+    }
```

What does this do? In other multiplatform libraries, I just see somethign like

```suggestion
    iosX64()
    iosArm64()
    iosSimulatorArm64()
```
Also, what about 

```
linuxX64()
linuxArm64()
macosX64()
macosArm64()
```
?